examples/04_llm_specific_tools/01_gpt5_apply_patch_preset.py

```
@@ -0,0 +1,47 @@
+"""
+
+import os
+
+from openhands.sdk import LLM, Agent, Conversation
+from openhands.tools.preset.gpt5 import get_gpt5_agent
+
+
+# Resolve API key from env
+api_key = os.getenv("LLM_API_KEY") or os.getenv("OPENAI_API_KEY")
+if not api_key:
+    raise SystemExit("Please set OPENAI_API_KEY or LLM_API_KEY to run this example.")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)